app/views/books/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= form_for @book do |form| %>
```

Comment:
I think if you use the `simple_form` translations you should probably use `simple_form_for` here.